src/order-requests/order-requests.service.ts

```
@@ -1,4 +1,234 @@
-import { Injectable } from '@nestjs/common';
+import { BadRequestException, Injectable, NotFoundException } from '@nestjs/common';
+import { OrderRequest, OrderRequestStatus, PrismaClient } from '@prisma/client';
+import { CreateOrderRequestDto } from './dto/create-order-request.dto';
+import { ApproveOrderRequestDto } from './dto/approve-order-request.dto';
+import { RejectOrderRequestDto } from './dto/reject-order-request.dto';
 
 @Injectable()
-export class OrderRequestsService {}
+export class OrderRequestsService {
+  deleteRequestAndItemsInTransaction(orderRequestId: string) {
```

ESLint

[error] 'orderRequestId' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)